Solution fails to launch if username/profile path contains spaces

Error I get while attempting to launch debugging:
```
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1250,5): error MSB3073: The command "del %LOCALAPPDATA%\Microsoft\Office\16.0\Wef\AppCommands /q" exited with code 1.
```

The directory exists, but when trying to run that command manually I found that `del` or `dir` on `%LOCALAPPDATA%\Microsoft\Office\16.0\Wef\AppCommands` fails ('not found') but I *can* `cd` to it.

--- a/WoodGroveExpenseTrends/WoodGrove Expense Trends.csproj
+++ b/WoodGroveExpenseTrends/WoodGrove Expense Trends.csproj
@@ -69,6 +69,6 @@
   </PropertyGroup>
   <Import Project="$(VSToolsPath)\SharePointTools\Microsoft.VisualStudio.SharePoint.targets" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup>
-    <PreBuildEvent>del %25LOCALAPPDATA%25\Microsoft\Office\16.0\Wef\AppCommands /q</PreBuildEvent>
+    <PreBuildEvent>del "%25LOCALAPPDATA%25\Microsoft\Office\16.0\Wef\AppCommands" /q</PreBuildEvent>
   </PropertyGroup>
 </Project>

